docs/docs/advanced/air-gap.md

```diff
@@ -1,142 +1,125 @@
 # Air-Gapped Environment
+Trivy needs to connect to the internet to download databases. If you are running Trivy in an air-gapped environment, or an tightly controlled network, this document will explain your options.
+In an air-gapped environment it is your responsibility to update the Trivy databases on a regular basis, so that the scanner can detect newly disclosed vulnerabilities.  
 
-Trivy can be used in air-gapped environments. Note that an allowlist is [here][allowlist].
+## Network requirements
+Trivy's Databases are distributed as OCI images via GitHub Container registry (GHCR):
 
-## Air-Gapped Environment for vulnerabilities
+- <https://ghcr.io/aquasecurity/trivy-db>
+- <https://ghcr.io/aquasecurity/trivy-java-db>
+- <https://ghcr.io/aquasecurity/trivy-checks>
 
-### Download the vulnerability database
-At first, you need to download the vulnerability database for use in air-gapped environments.
+If Trivy is running behind a firewall, you'll need to add the following urls to your allowlist:
 
-=== "Trivy"
+- `ghcr.io`
+- `pkg-containers.githubusercontent.com`
 
-    ```
-    TRIVY_TEMP_DIR=$(mktemp -d)
-    trivy --cache-dir $TRIVY_TEMP_DIR image --download-db-only
-    tar -cf ./db.tar.gz -C $TRIVY_TEMP_DIR/db metadata.json trivy.db
-    rm -rf $TRIVY_TEMP_DIR
-    ```
+The databases are pulled by Trivy using the [OCI Distribution](https://github.com/opencontainers/distribution-spec) specification, which is based on simple HTTPS protocol.
 
-=== "oras >= v0.13.0"
-    Please follow [oras installation instruction][oras].
+## Running Trivy in air-gapped environment
+In an air-gapped environment, you have to tell Trivy on every scan to not attempt to download the latest database files, otherwise the scan will fail. The following flags are relevant:
 
-    Download `db.tar.gz`:
+- `--skip-db-update` to skip updating the main vulnerability database.
+- `--skip-java-db-update` to skip updating the Java vulnerability database.
+- `--offline-scan` to scan Java applications without issuing API requests.
```

Comment:
We no longer need it and can delete it for now. But @DmitriyLewen suggested re-implementing sonatype API. We'll add it back after that.

Reply:
Also, users suggested that `--offline` enables all needed flags for air-gapped environments.

Reply:
https://github.com/aquasecurity/trivy/pull/7160/commits/4695eebf7a876ff18c9a56f07e4ef4c141dc7e10